mpa/modules/ov/models/mmdet/necks/mmov_ssd_neck.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from typing import Dict, List, Optional, Union
+
+import torch
+from mmdet.models.builder import NECKS
+from mmdet.models.necks.ssd_neck import SSDNeck
```

Comment:
The `mmdet` currently used in otx cannot be imported. If this is supported before mmpublic-det integration, it seems that this should be corrected.

Reply:
Indeed, I assumed public `mmdet`. I would copy `SSDNeck` from public `mmdet` for the time being.